base_site/h-series/index.html

```diff
@@ -1,7 +1,14 @@
 <!doctype html>
+<<<<<<< HEAD
 <title>H-series documentation placeholder</title>
 <body>
 <header>H-series documentation placeholder</header>
 <p>This should be replaced by the actual H-series documentation when it is built.</p>
+=======
+<title>H-Series documentation placeholder</title>
+<body>
+<header>H-Series documentation placeholder</header>
+<p>This should be replaced by the actual H-Series documentation when it is built.</p>
+>>>>>>> 875bed6 (rename subdir to h-series)
```

Comment:
```suggestion
<title>H-Series documentation placeholder</title>
<body>
<header>H-Series documentation placeholder</header>
<p>This should be replaced by the actual H-Series documentation when it is built.</p>
```

Not sure how the merge conflict markers ended up in here but at least they're easy to remove. :)